### PR TITLE
(doc) Update chocolateysetup.psm1 - fix grammar

### PR DESCRIPTION
**Pull request by hmemcpy**
"awhile" -> "a while"
per http://www.chicagomanualofstyle.org/qanda/data/faq/topics/Usage/faq0016.html :)

### CODE REVIEW

File: nuget/chocolatey/tools/chocolateysetup.psm1 [hunk 1]
```diff
@@ -465,7 +465,7 @@ param(
     $netFx4InstallTries += 1
 
     if (!(Test-Path $NetFx4Installer)) {
-      Write-Host "Downloading `'$NetFx4Url`' to `'$NetFx4Installer`' - the installer is 40+ MBs, so this could take awhile on a slow connection."
+      Write-Host "Downloading `'$NetFx4Url`' to `'$NetFx4Installer`' - the installer is 40+ MBs, so this could take a while on a slow connection."
```
Comment (ferventcoder) [inline nuget/chocolatey/tools/chocolateysetup.psm1]: :+1:

**Comment by ferventcoder** [comment]
Not sure why the builds failed (appveyor that is, I know why travis is failing)

**Comment by ferventcoder** [comment]
@hmemcpy can you review CONTRIBUTING and update your commit message summary to prefix `(doc)` please?

**Comment by hmemcpy** [comment]
Will do!

**Comment by ferventcoder** [comment]
Thanks!

**Comment by hmemcpy** [comment]
yay! :)

**Comment by synhershko** [comment]
For the record, that still doesn't make you open source contributor :P

**Comment by gep13** [comment]
:+1:

**Pull request merged**